application/txhandler.go

```diff
@@ -258,39 +258,67 @@ func (tm *txHandler) UTXOGetData(txn *badger.Txn, owner *objs.Owner, dataIdx []b
 	return tm.uHdlr.GetData(txn, owner, dataIdx)
 }
 
-func (tm *txHandler) GetValueForOwner(txn *badger.Txn, owner *objs.Owner, minValue *uint256.Uint256) ([][]byte, *uint256.Uint256, error) {
-	var u [][]byte
-	v := uint256.Zero()
-	utxoIDs, value, err := tm.uHdlr.GetValueForOwner(txn, owner, minValue)
-	if err != nil {
-		utils.DebugTrace(tm.logger, err)
-		return nil, nil, err
-	}
-	v, err = v.Clone().Add(v.Clone(), value.Clone())
-	if err != nil {
-		utils.DebugTrace(tm.logger, err)
-		return nil, nil, err
-	}
-	u = append(u, utxoIDs...)
-	if v.Lt(minValue) {
-		remainder, err := new(uint256.Uint256).Sub(minValue.Clone(), v.Clone())
+func (tm *txHandler) GetValueForOwner(txn *badger.Txn, owner *objs.Owner, minValue *uint256.Uint256, pt *objs.PaginationToken) ([][]byte, *uint256.Uint256, *objs.PaginationToken, error) {
+	const maxCount = 100
```

Comment:
Move maxCount into constants local or root level constants for easier documentation and upgradability.